dispatcher/templates/events_event_updated.jinja2

```diff
@@ -0,0 +1,81 @@
+
+<!DOCTYPE html>
```

Comment:
I just pick this file, but this applies to all: it's a lot of copy and paste, and I guess we'd like to better template this in the future?
It's always the same logic of title/body: 2 to 5 lines change between each file.

The second point, also for a follow-up task, is to remove the massive tableS (plural :sob: ) and use proper HTML to align the content. Our emails are not content heavy, so this makes a nice follow-up item. 

The two points are intertwined, but can be done in independent PRs: the HTML fix ideally would only be done once the HTML code has been factored in a single file.

Reply:
keep in mind (but you know, as per your comment in the PR thread) this is only a blind copy paste from the mailer templates that were in elixir

Reply:
yup, I did some research after posting this.
I don't expect action here, but I'd leave it open, until tracked in Jira.